page_objects/Base_Page.py

```diff
@@ -133,10 +132,13 @@ def set_test_run_id(self,test_run_id):
     def register_tesults(self):
         "Register Tesults with Page"
         self.tesults_flag = True
+        from utils import Tesults # pylint: disable=import-error,import-outside-toplevel
+        self.tesult_objects = Tesults
```

Comment:
Can you rename this variable to `self.testult_object`?

Reply:
The name is "Tesults" not "testult". May I know the reason to change the name ?

Reply:
Sorry. I meant `self.tesult_object` instead of `self.tesult_objects`

Reply:
done!